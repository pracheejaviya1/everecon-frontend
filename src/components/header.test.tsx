import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';

describe('Header', () => {
  it('Renders Header', () => {
    const tree = renderer.create(<Header headerText='Default Test' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
