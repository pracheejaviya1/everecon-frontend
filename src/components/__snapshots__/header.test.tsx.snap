// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders Header 1`] = `
<h1
  className="m-2 font-bold"
>
  Default Test
</h1>
`;
